### ISSUE TITLE
small issues when consuming repl-tooling

### ISSUE THREAD

**Issue by awb99** [mauricioszabo/repl-tooling#79]
1. /src/pinkgorilla/ui/pinkie.cljs is being published to the uberjar; this does not let me use repl-tooling together with gorilla-ui. I just removed it in my local build (lein install) and it works fine.

2. please release on clojars also.

3. shadow-cljs warnings:
```
[2020-06-12 20:54:37.509 - INFO] filename violation for ns cljs-blob, got: cljs-blob.cljs expected: cljs_blob.cljs (or .cljc)
[2020-06-12 20:54:37.513 - INFO] filename violation for ns cljs-cmd-wrap, got: cljs-cmd-wrap.cljs expected: cljs_cmd_wrap.cljs (or .cljc)
```

culprits are:
```
src/repl_tooling/integrations/connection.cljs
2:  (:require-macros [repl-tooling.repl-client.clj-helper :refer [cljs-blob-contents]

src/repl_tooling/repl_client/clj_helper.clj
20:(defmacro cljs-blob-contents []
21:  (slurp (io/resource "cljs-blob.cljs")))
```

**Comment by mauricioszabo** [comment]
(Sorry, I missed this issue)

Yes, I was experimenting with namespaces, forgot to remove this code. Will remove it on next version :).

Also, it's being published on Clojars now, but as a SNAPSHOT version only. Will do a version bump soon-ish :)

**Comment by mauricioszabo** [comment]
I believe these are fixed on version 0.5.0 :)
